app/scripts/modules/core/src/search/global/globalSearch.less

```diff
@@ -28,7 +28,7 @@ global-search {
     right: 0;
     margin: 2px 19px;
     border-radius: 0;
-    z-index: 10000;
+    z-index: 10001;
```

Comment:
🤮

Reply:
yeah... unfortunately using the critical variable (aka `z-index: 1000`) was not cutting it.

Reply:
Wondering... can you do:
```  
z-index: calc(var(--layer-critical) + 1);
```
still not good but perhaps easier to understand the intent?